profiles/common/modules/features/ec_embedded_video/README.md

```diff
@@ -0,0 +1,46 @@
+The "EC Embedded Video" feature allows to embed videos from YouTube, Vimeo, AV Portal or Dailymotion in "File" fields or in WYSIWYG
+fields.
+
+Table of content:
+=================
+- [Installation](#installation)
+- [Proposed features](#features)
+
+# Installation
+
+The feature is proposed through a Drupal custom module and is available with the NextEuropa platform.
+
+They are 2 ways to activate the feature:
+1. Like any Drupal modules; via the "Modules" admin page or with the Drush command);
+2. Like a "Feature set" through the Feature set admin page (path: admin/structure/feature-set_en).<br />
+Then, Enable the "Embedded videos" feature and click on the "Validate" button.
+
+[Go to top](#table-of-content)
+
+# Proposed features
+
+It allows inserting a video URL of a video hosted by YouTube, Vimeo, AV Portal or Dailymotion in "File" fields or 
+in WYSIWYG field.
+
+To do so, you just have to copy the URL from the site and paste it in field accessible via the media browser modal under 
+the "Web" tab.
+
+The feature proposed also 2 display modes for videos from YouTube, Vimeo, AV Portal or Dailymotion and that are inserted in WYSIWYG
+fields:
+1. The one supplied by modules managing every video of these types (AV Portal Video, Dailymotion Video,  Vimeo Video, YouTube Video).<br />
+It embeds videos in an HTML wrapper (DIV tags).
+2. The one supplied by the feature itself for these types (AV Portal Video without wrapper, Dailymotion Video without wrapper, 
+Vimeo Video without wrapper, YouTube Video without wrapper).<br />
+It displays the video player with any HTML wrapper (no DIV tags).
+
+The module uses by default the second one for all videos but you can change easily the display in 2 different ways:
```

Comment:
This part of the documentation is a bit confusing because the "Default" display type doesn't use the second mode (custom), but uses the default one instead.

I think the documentation should say that the second (custom) option is the default for the "WYSIWYG" display mode, but that any display configuration can use it in admin/structure/file-types/manage/video/file-display/{display-type}.